astropy/units/photometric.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+This module defines magnitude zero points and related photometric quantities.
+"""
+
+
+import numpy as _numpy
+from .core import UnitBase, def_unit
+
+from ..constants import si as _si
+from . import cgs, si, astrophys
+
+
+_ns = globals()
+
+def_unit(['Bol', 'L_bol'], _si.L_bol0, namespace=_ns, prefixes=False,
+         doc="Luminosity corresponding to absolute bolometric magnitude zero")
+def_unit(['bol', 'f_bol'], _si.L_bol0 / (4 * _numpy.pi * (10.*astrophys.pc)**2),
+         namespace=_ns, prefixes=False, doc="Irradiance corresponding to "
+         "appparent bolometric magnitude zero")
+def_unit(['ABflux'], 3631e-23 * cgs.erg * cgs.cm**-2 / si.s / si.Hz,
```

Comment:
To keep the round-tripping working, which was something that @pllim was quite keen on before, we could include `AB` and `ST` as equivalent names in the list...

Alternatively, we need to continue to jump through some hoops in `logarithmic.py`.